articles/azure-app-configuration/quickstart-resource-manager.md

```diff
@@ -28,6 +28,13 @@ If your environment meets the prerequisites and you're familiar with using ARM t
 
 If you don't have an Azure subscription, create a [free account](https://azure.microsoft.com/free/?WT.mc_id=A261C142F) before you begin.
 
+## Authorization
+
+Accessing the key-value data used in ARM templates requires an ARM role, such as contributor or owner. The access via one of the Azure App Configuration [data plane roles](concept-enable-rbac.md) is currently not supported. Support may be added in the future.
```

Comment:
Wording seems a bit inconsistent with the note below.

Perhaps "Accessing key-value data inside an ARM template requires..."

Also "The access" in the second sentence should just be "Access".

Reply:
I'm trying to find out if we use ARM role anywhere else or if we use management plane role.

Reply:
I found that we do go a little more in depth [here](https://docs.microsoft.com/en-us/azure/azure-app-configuration/howto-disable-access-key-authentication?tabs=portal#permissions-for-allowing-or-disallowing-access-key-authentication), but perhaps not necessary for this note.

Reply:
Updated. Thanks, Jimmy! Yes, I copied the "ARM role" wording from the doc you linked :)